tests/system/providers/google/cloud/automl/example_automl_video_intelligence_classification.py

```diff
@@ -73,73 +72,88 @@
     schedule="@once",
     start_date=datetime(2021, 1, 1),
     catchup=False,
-    user_defined_macros={"extract_object_id": extract_object_id},
-    tags=["example", "automl", "video-clss"],
+    tags=["example", "automl", "video", "classification"],
 ) as dag:
     create_bucket = GCSCreateBucketOperator(
         task_id="create_bucket",
-        bucket_name=DATA_SAMPLE_GCS_BUCKET_NAME,
+        bucket_name=VIDEO_GCS_BUCKET_NAME,
         storage_class="REGIONAL",
-        location=GCP_AUTOML_LOCATION,
+        location=REGION,
     )
 
     move_dataset_file = GCSSynchronizeBucketsOperator(
         task_id="move_dataset_to_bucket",
         source_bucket=RESOURCE_DATA_BUCKET,
         source_object="automl/datasets/video",
-        destination_bucket=DATA_SAMPLE_GCS_BUCKET_NAME,
+        destination_bucket=VIDEO_GCS_BUCKET_NAME,
         destination_object="automl",
         recursive=True,
     )
 
-    create_dataset = AutoMLCreateDatasetOperator(
-        task_id="create_dataset", dataset=DATASET, location=GCP_AUTOML_LOCATION
+    create_video_dataset = CreateDatasetOperator(
+        task_id="video_dataset",
+        dataset=VIDEO_DATASET,
+        region=REGION,
+        project_id=PROJECT_ID,
     )
-
-    dataset_id = cast(str, XComArg(create_dataset, key="dataset_id"))
-    MODEL["dataset_id"] = dataset_id
-
-    import_dataset = AutoMLImportDataOperator(
-        task_id="import_dataset",
-        dataset_id=dataset_id,
-        location=GCP_AUTOML_LOCATION,
-        input_config=IMPORT_INPUT_CONFIG,
+    video_dataset_id = create_video_dataset.output["dataset_id"]
+
+    import_video_dataset = ImportDataOperator(
+        task_id="import_video_data",
+        dataset_id=video_dataset_id,
+        region=REGION,
+        project_id=PROJECT_ID,
+        import_configs=VIDEO_DATA_CONFIG,
     )
 
-    MODEL["dataset_id"] = dataset_id
-
-    create_model = AutoMLTrainModelOperator(task_id="create_model", model=MODEL, location=GCP_AUTOML_LOCATION)
-    model_id = cast(str, XComArg(create_model, key="model_id"))
-
-    delete_model = AutoMLDeleteModelOperator(
-        task_id="delete_model",
-        model_id=model_id,
-        location=GCP_AUTOML_LOCATION,
-        project_id=GCP_PROJECT_ID,
+    # [START how_to_cloud_create_video_classification_training_job_operator]
+    create_auto_ml_video_training_job = CreateAutoMLVideoTrainingJobOperator(
+        task_id="auto_ml_video_task",
+        display_name=VIDEO_DISPLAY_NAME,
+        prediction_type="classification",
+        model_type="CLOUD",
+        dataset_id=video_dataset_id,
+        model_display_name=MODEL_DISPLAY_NAME,
+        region=REGION,
+        project_id=PROJECT_ID,
+    )
+    # [END how_to_cloud_create_video_classification_training_job_operator]
+
+    delete_auto_ml_video_training_job = DeleteAutoMLTrainingJobOperator(
```

Comment:
Should we make it part of the doc just like `how_to_cloud_create_video_classification_training_job_operator`?

Reply:
Added